### DISCUSSION TITLE
[Collapsible] Disable initial "open" animation

### DISCUSSION THREAD

**Discussion by rkumorek** [radix-ui/primitives#976] [Help]
Hey,

I am keen on using Collapsible component with height animation. Currently have this which is basically example copied from docs.

```js
const CollapsibleContent = styled(CollapsiblePrimitive.Content, {
    overflow: "hidden",
    '&[data-state="open"]': {
        animation: `${keyframes({
            from: { height: 0 },
            to: { height: "var(--radix-collapsible-content-height)" },
        })} 200ms ease-out`,
    },
    '&[data-state="closed"]': {
        animation: `${keyframes({
            from: { height: "var(--radix-collapsible-content-height)" },
            to: { height: 0 },
        })} 200ms ease-out`,
    },
});
```

What happens is when the component is mounted it's animated in which I would like to avoid. What is the best way of disabling initial "open" animation on first render?

Thanks

**Comment by jjenzz** [comment]
I've created a PR to prevent mount animation when open by default here https://github.com/radix-ui/primitives/pull/977

**Reply by rkumorek** [reply]
Thank you, that looks great!

Apologies for duplicate discussion (feel free to delete this one if it's possible in favour of [934](https://github.com/radix-ui/primitives/discussions/934)).
